packages/devtools_app/lib/src/inspector/inspector_tree_controller.dart

```diff
@@ -616,8 +622,9 @@ class InspectorTreeController extends Object
     if (searchPreviousMatches) {
       final previousMatches = searchMatches.value;
       for (final previousMatch in previousMatches) {
-        if (previousMatch.node.diagnostic.searchValue
-            .caseInsensitiveContains(search)) {
+        if (cachedRows.contains(previousMatch) &&
```

Comment:
will this ever be false? If a row was a previous match, wouldn't it always be in cachedRows?